fbw-a380x/src/systems/systems-host/systems/FlightWarningSystem/FwsCore.ts

```diff
@@ -2593,24 +2687,31 @@ export class FwsCore implements Instrument {
 
     this.landAsapRed.set(
       !this.aircraftOnGround.get() &&
-        (this.fireButton1.get() ||
-          this.eng1FireTest.get() ||
-          this.fireButton2.get() ||
-          this.eng2FireTest.get() ||
-          this.fireButtonAPU.get() ||
-          this.apuFireTest.get() ||
+        (this.apuFireDetected.get() ||
+          this.eng1FireDetected.get() ||
+          this.eng2FireDetected.get() ||
+          this.eng3FireDetected.get() ||
+          this.eng4FireDetected.get() ||
+          this.mlgFireDetected.get() ||
           this.emergencyGeneratorOn.get() ||
           (this.engine1State.get() === 0 && this.engine2State.get() === 0) ||
           (this.greenLP.get() && this.yellowLP.get()) ||
           (this.yellowLP.get() && this.blueLP.get()) ||
           (this.greenLP.get() && this.blueLP.get())),
     );
 
+    // Is this needed?
```

Comment:
Good question, don't know why it was needed on the A32NX in the first place. I'd propose we keep it there and if it causes any trouble we can remove it

Reply:
Sounds good and agreed, I've removed the comment.